### ISSUE TITLE
Characters start off with Masks that are really noses???  And hats that are really hair.  That's weird.

### ISSUE THREAD

**Issue by RayR10** [mdotstrange/NightmarePuppeteerPublic#445]
When I first started using this, I couldn't figure out why all my random characters were wearing the same masks and headgear.  I was clicking on "Delete All the Props on this Actor", but every head I generated still had the dumb mask and headgear.  Also, I was making sure I was  generating a random character **without** props. 

I FINALLY figured out that I had to go into the **second Edit Head menu** and delete the nose. which isn't really a nose, but a mask apparently.  LOL.  

And then the headgear was actually the character's hair apparently.  So yeah.  Massive confusion there.

Kinda wish there was a Scale Slider for the body, like there is for the Head.  That's be nice.  You could make tiny people or big monsters.  I supposed that'd be tough to do with the underlying bone setup.

**Comment by mdotstrange** [comment]
I believe you can scale the actors with a console command- press Shift + Esc to bring up the console and search scale etc- their ik gets funky tho- there's "ScaleMainActor" + you can scale all their body parts separately

**Comment by mdotstrange** [comment]
"That's weird." 

Its Nightmare Puppeteer- its weird by design. (thumbs up)

**Comment by RayR10** [comment]
Thanks for the info about Scaling.  It's an amazing program.  I'll just have to stumble through the growing pains of learning it.  I was looking for a program like the old MS Movie Maker.  I did an exhaustive Google search about 2 years ago and came up dry.  Then I stumbled on this last week.  Never knew it existed.  I never thought to look on Steam (I'm not much of a gamer).  Glad I found this.  I look forward to making some cool little movies.
